manage.py

```diff
@@ -55,6 +54,22 @@ def __init__(self, args: List[str]):
             os.environ.setdefault("DEBUG", "true")
             print("Starting in debug mode.")
 
+    @staticmethod
+    def parse_db_url(db_url: str) -> SplitResult:
+        """Validate and split the given databse url."""
+        db_url_parts = urlsplit(db_url)
+        if not all((
+            db_url_parts.hostname,
+            db_url_parts.port,
+            db_url_parts.username,
+            db_url_parts.password,
+            db_url_parts.path
+        )):
+            raise ValueError(
+                "The DATABASE_URL environment variable is not a valid PostgreSQL database URL."
```

Comment:
The function is technically not restricted to only checking `DATABASE_URL` despite what the error says, but whatever; it's a minor detail.